netci.groovy

```diff
@@ -78,8 +78,37 @@ static void addVsiArchive(def myJob) {
   Utilities.addArchival(myJob, archiveSettings)
 }
 
-addVsiArchive(newVsiJob)
+// ISSUE: Temporary until a full builder for multi-scm source control is available.
+// Replace the scm settings with a multiScm setup.  Note that this will not work for PR jobs
```

Comment:
@mmitche Is there any way this can be enabled for PR jobs as well? We would eventually want to enable the VSI tests to run for every project system PR.